apps/src/lib/util/audioApi.js

```diff
@@ -109,6 +115,53 @@ export const commands = {
     } else {
       Sounds.getSingleton().stopAllAudio();
     }
+  },
+  playSpeech(opts) {
```

Comment:
Similar comment to above. Since this is unique to speech, perhaps it should be in its own location with the other speech-specific functions

Reply:
Offline conversation: I hadn't realized this was the droplet function. Instead of moving this whole thing to its own file, it probably makes more sense to move the SpeechConfig and SpeechSynthesizer sections to their own file but leave the input validation here.